### ISSUE TITLE
Versão 62 com Janela de Propaganda

### ISSUE THREAD

**Issue by r4fazzawa** [renatofmmaia/bomb-bot#166]
Bom dia !!
Alguém conseguiu programar o bot para fechar essa Janela de propaganda do Bomb logo após o Login?

![image](https://user-images.githubusercontent.com/113998376/191270061-73996d5c-d379-486a-8b0f-d319354ce75c.png)

**Comment by morfer** [comment]
Tem que alterar o arquivo "module\bombScreen.py"

Depois do bloco:

```
                logger_translated("sigin wallet", LoggerEnum.BUTTON_CLICK)
                if not click_when_target_appears("button_connect_wallet_sign"):
                    refresh_page()
                    continue
```

Adiciona esse bloco de código:

```
  logger_translated("fecha popup", LoggerEnum.BUTTON_CLICK)
                if not click_when_target_appears("buttun_x_close"):
                    refresh_page()
                    continue
```

**Comment by megamandev** [comment]
esquece removeram o modo caça tesouro

**Comment by megamandev** [comment]
alguem tem detalhes informaçoes sobre essa update 63, sei que removeram o modo caça tesouro, postem aqui os detalhes

**Comment by morfer** [comment]
Sim, removeram o treasure hunt, mas o bot continua funcionando para o modo amazon.
Eles removeram o treasure hunt, pq é um modo que não gira a economia do jogo. No modo amazon, vc sempre vai ter que renovar os heróis, escudos, etc. Ou seja, vc tem que colocar dinheiro no jogo. Isso ajuda a sustentabilidade do projeto a longo prazo.
Agora, se vale a pena, é outra história.. rss

**Comment by megamandev** [comment]
o reset roi então acabou ?

**Comment by morfer** [comment]
O ícone ainda está lá, e estão falando que nerfaram o modo amazon.
Como eu não tenho herói S pra jogar o modo amazon, não sei dizer se o nerf vai continuar ou não.

**Comment by megamandev** [comment]
tamben nao tenho heroi S, tenho uma casa mas nao sei como programar esse bot pra clicar pra enviar pra casa
